test/Core.Test/AutoFixture/GlobalSettingsFixtures.cs

```diff
@@ -15,13 +19,34 @@ public object Create(object request, ISpecimenContext context)
             throw new ArgumentNullException(nameof(context));
         }
 
-        var pi = request as ParameterInfo;
         var fixture = new Fixture();
 
-        if (pi == null || pi.ParameterType != typeof(Bit.Core.Settings.GlobalSettings))
+        if (request is not ParameterInfo pi)
+        {
             return new NoSpecimen();
+        }
+
+        if (pi.ParameterType == typeof(Bit.Core.Settings.GlobalSettings))
+        {
+            return GlobalSettingsFactory.GlobalSettings;
+        }
+
+        if (pi.ParameterType == typeof(IDataProtectionProvider))
+        {
+            var dataProtector = new Mock<IDataProtector>();
+            dataProtector
+                .Setup(d => d.Unprotect(It.IsAny<byte[]>()))
+                .Returns<byte[]>(data => Encoding.UTF8.GetBytes("P|" + Encoding.UTF8.GetString(data))); // I THINK?
```

Comment:
⚠️ Use the constant.

Is this now working as expected (thus the comment)?

Reply:
The `I THINK` is more of a question to however you want it to work. It doesn't actually protect anything but it's a test so I think that is fine and it offers a hook into it where you could evaluate whether or not it's protected.